Release/include/cpprest/ws_msg.h

```diff
@@ -64,6 +64,12 @@ class websocket_outgoing_message
     /// This is useful when the client side wants to check whether the server is alive.
     /// </summary>
     void set_pong_message() { this->set_message_pong(); }
+
+    /// <summary>
+    /// Sets a the outgoing message to be a ping message.
+    /// This is useful when the client side wants to check whether the server is alive.
+    /// </summary>
+    void set_ping_message() { this->set_message_ping(); }
```

Comment:
Suggestion: add optional application-data argument, i.e. `const std::string& data = ""`, since ping/pong messages _may_ have bodies up to ~125 bytes. If so, should also be added to `set_pong_message`.

Reply:
Added string data param